src/process.rs

```diff
@@ -183,30 +179,43 @@ pub async fn kinesis_logs(
         .clone()
         .unwrap_or_else(|| "NO SUBSYSTEM NAME".to_string());
     let v = &kinesis_message.0;
```

Comment:
I would try removing the `&` here, ant then we should be able to avoid the  `v.clone()` on lines 186 and 195